Commands/InteractionServiceHandler.cs

```diff
@@ -52,11 +53,11 @@ public static async Task RegisterModulesAsync()
 
 			await _interactionService.RegisterCommandsGloballyAsync();
 			_client!.InteractionCreated += OnInteractionCreatedAsync;
-			await Program.LogAsync("InteractionServiceManager", "Modules registered successfully", LogSeverity.Info);
+			Log.Information("Modules registered successfully, InteractionServiceManager");
```

Comment:
Same situation as above
It could also be simplified to just "Registered modules", but I'm not sure if the "Modules registered successfully" isn't better, as it's clearly indicating a success

```suggestion
			Log.Information("Registered modules");
```
or
```suggestion
			Log.Information("Modules registered successfully");
```